generate/generate.go

```diff
@@ -1062,6 +1074,23 @@ func mapType(t string) string {
 	}
 }
 
+func mapResponseType(t string) string {
+        switch t {
+        case "boolean":
+                return "bool"
+        case "short", "int", "integer", "long":
+                return "int64"
```

Comment:
Ah, yes, I meant to change that so only long returned int64! But I found an compile error when putting that case statement in the existing mapType. I'll look again and see if there's an easy way to correct those errors